packages/nextra/package.json

```diff
@@ -86,9 +90,15 @@
     "slash": "^3.0.0"
   },
   "peerDependencies": {
+    "mermaid": ">=9.1.5",
     "next": ">=9.5.3",
     "react": ">=16.13.1",
-    "react-dom": ">=18.2.0"
+    "react-dom": ">=16.13.1"
+  },
+  "peerDependenciesMeta": {
+    "mermaid": {
+      "optional": true
+    }
```

Comment:
making this package optional so users that don't use mermaid will not download this package